### PR TITLE
Pallindrome number

### PR DESCRIPTION
**Pull request by Amit366**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

Article for Pallindrome number

### Checklist

- [ ] I've read the contribution guidelines.
- [ ] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes: #65

### CODE REVIEW

**Comment by Amit366** [comment]
@HarshCasper please review

File: docs/mathematics/pallindrome-number.mdx [hunk 1]
```diff
@@ -0,0 +1,133 @@
+---
+title: Pallindrome Number
+slug: /mathematics/pallindrome-number
```
Comment (HarshCasper) [inline docs/mathematics/pallindrome-number.mdx:3]: There is a typo in spelling of palindrome.

File: docs/mathematics/pallindrome-number.mdx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+---
+
+## Introduction
+
+A palindromic number (also known as a numeral palindrome or a numeric palindrome) is a number (such as 16461) that remains the same when its digits are reversed. The term palindromic is derived from palindrome, which refers to a word (such as rotor or racecar) whose spelling is unchanged when its letters are reversed.
```
Comment (HarshCasper) [inline docs/mathematics/pallindrome-number.mdx:8]: Plagiarized from [here](https://en.wikipedia.org/wiki/Palindromic_number)

File: docs/mathematics/pallindrome-number.mdx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+### Quick Example :
+
+Consider a number `17371`. If this is reversed then again we get `17371` which is same as the original number. Hence `17371` is a pallindrome number.
+
+Consider a number `1243`. If this is reversed then we get `3421` which is not same as the original number. Hence `1243` is not a pallindrome number.
+
+All `single digit` integers are pallindrome numbers.
+
+Some pallindrome numbers are: `0`, `1`, `2`, `3`, `4`, `5`, `6`, `7`, `8`, `9`, `11`, `22`, `33`, `44`, `55`, `66`, `77`, `88`, `99`, `101`, `111`, .....
```
Comment (HarshCasper) [inline docs/mathematics/pallindrome-number.mdx:22]: Keep in triple backticks

File: docs/mathematics/pallindrome-number.mdx [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+## Explanation
+
+A number `pqr` is said to be pallindrome, if and only if it's reverse is also `pqr`. This means the reverse of `pqr` is `rqp` which must be equal to it's original number `pqr`.
+
+### Quick Example :
+
+Consider a number `17371`. If this is reversed then again we get `17371` which is same as the original number. Hence `17371` is a pallindrome number.
+
+Consider a number `1243`. If this is reversed then we get `3421` which is not same as the original number. Hence `1243` is not a pallindrome number.
```
Comment (HarshCasper) [inline docs/mathematics/pallindrome-number.mdx:18]: Keep both of them as bullet points

File: docs/mathematics/pallindrome-number.mdx [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+All `single digit` integers are pallindrome numbers.
```
Comment (HarshCasper) [inline docs/mathematics/pallindrome-number.mdx:20]: Keep it as a note

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Please fix the merge conflicts

File: docs/mathematics/pallindrome-number.mdx (hunk 3 above)
Comment (Amit366) [reply docs/mathematics/pallindrome-number.mdx:22]: Which part? @HarshCasper

**Comment by Amit366** [comment]
@HarshCasper done

File: docs/mathematics/palindrome-number.mdx [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
+A number `pqr` is said to be palindrome, if and only if it's reverse is also `pqr`. This means the reverse of `pqr` is `rqp` which must be equal to it's original number `pqr`.
+
+### Quick Example :
+
+* Consider a number `17371`. If this is reversed then again we get `17371` which is same as the original number. Hence `17371` is a palindrome number.
+
+* Consider a number `1243`. If this is reversed then we get `3421` which is not same as the original number. Hence `1243` is not a palindrome number.
+
+**Note:** All `single digit` integers are palindrome numbers.
```
Comment (HarshCasper) [inline docs/mathematics/palindrome-number.mdx:20]: Keep `single digit` in bold.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Tiny change required. Sending it off for a peer review.

**Comment by HarshCasper** [comment]
Please fix the merge conflicts so that I can merge this.

**Comment by Amit366** [comment]
@HarshCasper done

**Comment by HarshCasper** [comment]
@Amit366 Let me fix the build for you.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**